ppr-ui/src/mixins/date-mixin.ts

```diff
@@ -15,7 +15,7 @@ export default class DateMixin extends Mixins(CommonMixin) {
    * @returns a promise to return a Date object
    */
   async getServerDate (): Promise<Date> {
-    const input = `${window.location.origin}/${process.env.VUE_APP_PATH}/`
+    const input = `${window.location.origin}${process.env.VUE_APP_PATH}/`
```

Comment:
I'm not entirely sure if this method is used anywhere, potentially by a common or shared component?
Either way, no harm in updating!